rules/linkedin-connect-with-people/rules.md

```diff
@@ -6,30 +6,52 @@ guid: 12a2e1c9-fc4e-4c10-9c4f-f11ed665e668
 uri: linkedin-connect-with-people
 created: 2024-04-29T15:00:00.000Z
 authors: 
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan/
+  - title: Brady Stroud
+    url: https://ssw.com.au/people/brady-stroud/
+  - title: Tylah Kapa
+    url: https://ssw.com.au/people/tylah-kapa/
   - title: Hugo Pernet
     url: https://ssw.com.au/people/hugo-pernet/
 related:
   - successful-reachouts
+  - weekly-client-love
 ---
 
-Networking platforms like LinkedIn are essential for connecting with professionals in your industry. With numerous connections, remembering interaction specifics can be challenging. To foster meaningful relationships, personalize your connection requests!
+Networking platforms like LinkedIn are essential for connecting with professionals in your industry. As your network grows, recalling the specifics of each interaction becomes increasingly challenging. To foster meaningful relationships and maintain a context for each contact, personalize your connection requests!
 
 <!--endintro-->
 
-1. On the LinkedIn mobile app, avoid using the default "Connect" or "Message" buttons. Instead, use the "ellipsis | Personalize invite" button to craft a custom message. On the LinkedIn website, click "Connect | Add a note"
+1. On the LinkedIn mobile app - avoid using the default "Connect" or "Message" buttons. Instead, use the "ellipsis | Personalize invite" button to craft a custom message.
 
+   On the LinkedIn website - click "Connect | Add a note"
+
+::: good
 ![Figure: Use "Personalize invite" instead of "Connect" or "Message"](personnalize-invite.jpeg)
+:::
```

Comment:
I think you also need a bad example here

Reply:
Don't know what to add there, the screenshot actually shows what's bad (clicking connect directly) and what's good (using personalize invite)